examples/core/3d-texture/app.js

```diff
@@ -152,12 +152,27 @@ class AppAnimationLoop extends AnimationLoop {
       }
     });
 
-    return {cloud, mvpMat, viewMat, statsWidget};
+    const memWidget = new StatsWidget(statsManager.get('Memory Usage'), {
```

Comment:
Side note:

My proposal is that we cut this widget from the individual examples and add it to the website.

That way we get the benefits in all demos without cluttering up every app.

I made a big push on the website in the pending scenegraph split PR, should be able to go through how it would work once that lands.